test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
```

Comment:
>  I think the day of managing gem dependencies manually, from a set of system gems, is gone

I agree.
"bundle exec ruby test/basic_test.rb (doesn't work on master)." should work at least.